src/services/utility.spec.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import * as u from './utility';
+
+describe('utility assign', () => {
+
+	function testObject() {
+		return {
+			foo: 1,
+			bar: {
+				baz: 'hello'
+			}
+		};
+	}
+
+	it('pass null should not change target', () => {
```

Comment:
Consider to have more fluent statements: `it('should not change target when null is passed', ...` :)